pom.xml

```diff
@@ -68,6 +69,16 @@
                 <module>driver-cqld4</module>
             </modules>
         </profile>
+        <profile>
+            <id>with-mongodb</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
```

Comment:
This needs to be set to true to work with the module inclusion above, in which case the lower module inclusion also applies. I suggest we just make it on by default now that it is building and working.

Reply:
Updated. :ok_hand: